packages/slider/package.json

```diff
@@ -38,27 +35,29 @@
   "devDependencies": {
     "@babel/cli": "^7.16.7",
     "@babel/core": "^7.16.7",
-    "@priceline/babel-preset": "workspace:*",
     "@priceline/eslint-config": "workspace:*",
+    "@priceline/heft-component-rig": "workspace:*",
+    "@priceline/storybook-preset": "workspace:*",
     "@reach/component-component": "^0.16.0",
     "@rushstack/eslint-patch": "^1.1.0",
+    "@rushstack/heft": "^0.44.2",
+    "@storybook/addon-actions": "^6.4.10",
+    "@storybook/react": "^6.4.10",
     "@testing-library/jest-dom": "^5.15.0",
     "@testing-library/react": "^12.1.2",
-    "@wojtekmaj/enzyme-adapter-react-17": "^0.6.5",
-    "cross-env": "^7.0.3",
-    "enzyme": "^3.11.0",
     "eslint": "^8.6.0",
     "jest": "^26.6.3",
     "jest-standard-reporter": "^2.0.0",
     "jest-styled-components": "^6.3.4",
-    "npm-run-all": "^4.1.5",
     "pcln-design-system": "workspace:*",
-    "rc-tools": "^8.4.5",
-    "rc-trigger": "^2.6.5",
+    "rc-slider": "^10.0.0-alpha.4",
```

Comment:
should this be the alpha version?

Reply:
that's what they're publishing as their `latest`, so I assume so